services/app-api/serverless.yml

```diff
@@ -67,8 +49,8 @@ provider:
     restApi: true
   iam:
     role:
-      path: ${self:custom.iamPath}
-      permissionsBoundary: ${self:custom.iamPermissionsBoundaryPolicy}
+      path: ${ssm:/configuration/${self:custom.stage}/iam/path~true, ssm:/configuration/default/iam/path~true, "/"}
+      permissionsBoundary: ${ssm:/configuration/${self:custom.stage}/iam/permissionsBoundaryPolicy~true, ssm:/configuration/default/iam/permissionsBoundaryPolicy~true, ""}
```

Comment:
Since we no longer use path or permissions boundary in our cloudformation code (its handled by plugins), theres no need to set it to a custom variable.  There never really strictly was a reason we had to set it to a custom variable.  But now since it is exclusively used in the provider block, we just set it there directly.